release_notes.md

```diff
@@ -8,29 +8,15 @@
   - Fix thread context classloader for middleware chain
 - Update Java Worker Version to [2.15.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.15.0):
   - Update grpc-protobuf to 1.64.0 and application insights agent version to 3.5.2
-- Marking SyncTriggers [RequiresRunningHost]. (#10233)
-- Defaulting SwtAuthenticationEnabled to False (#10195)
-- Adding runtime site name to valid JWT audiences (slot scenarios) (#10185)
 - Update Python Worker Version to [4.30.3](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.30.3)
-- Language worker channels will not be started during placeholder mode if we are in-process (#10161)
-- Ordered invocations are now the default (#10201)
-- Add IsDisabled property to worker description and skip if the value (#10250)
-- Fixed incorrect function count in the log message.(#10220)
-- Migrate Diagnostic Events to Azure.Data.Tables (#10218)
-- Sanitize worker arguments before logging (#10260)
-- Fix race condition on startup with extension RPC endpoints not being available. (#10282)
-- Adding a timeout when retrieving function metadata from metadata providers (#10219)
-- Upgraded the following package versions (#10288):
-  - `Microsoft.Azure.WebJobs` updated to 3.0.41
-  - `Microsoft.Azure.WebJobs.Host.Storage` updated to 5.0.1
-  - `Microsoft.Extensions.Azure` updated to 1.7.1
-  - `Azure.Storage.Blobs` updated to 12.19.1
-- [in-proc] Updating FunctionsNetHost (dotnet isolated worker) to 1.0.11 (#10379)
 - Upgraded the following package versions (#10326):
   - `Azure.Security.KeyVault.Secrets` updated to 4.6.0
   - `System.Format.Asn1` updated to 6.0.1
 - Update PowerShell 7.2 worker to [4.0.4020](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.4020)
 - Update PowerShell 7.4 worker to [4.0.4021](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.4021)
-- Trim FunctionsNetHost artifacts (#10364)
 - Resolved thread safety issue in the `GrpcWorkerChannel.LoadResponse` method. (#10352)
 - Worker termination path updated with sanitized logging (#10397)
+- [in-proc] Avoid redundant DiagnosticEvents error message (#10396)
+- Fixing thread safety issue in the GrpcWorkerChannel.LoadResponse method (#10363) (#10400)
+- Updating DotNetIsolatedNativeHost package to 1.0.11 (#10380)
```

Comment:
Is this not already part of [4.35.4](https://github.com/Azure/azure-functions-host/releases/tag/v4.35.4)?

Reply:
@kshyju These were not included in 4.35.4 release notes

Reply:
Sorry! What do you mean they were not included? It is listed in the release notes I linked above. You can also see the commit here in 4.35.4 tag - https://github.com/Azure/azure-functions-host/blob/v4.35.4/src/WebJobs.Script/WebJobs.Script.csproj#L56

Reply:
@kshyju I appreciate your guidance on this matter. However, as mentioned, we typically rely on release notes for information. Unfortunately, the details from version 4.35.4 were not included in the release notes, even though they were present in the code.